src/components/transaction-dialog-select/transaction-dialog-select.js

```diff
@@ -0,0 +1,58 @@
+import template from "./transaction-dialog-select.html";
```

Comment:
make this component 'dump' -- it should be just select (cal it smth like CustomSelectComponent)-- it should not know anything about transactions or tags -- it should have a prop `items` -- that should be a list of strings -- and again it should not know anything about these items -- it should just use them